client/views/components/generic/input-text/InputText.jsx

```diff
@@ -82,7 +82,7 @@ class InputText extends Component {
     const { date, focused, value } = this.state;
 
     return (
-      <div className={`input input-text ${className} ${validationStatus || ''}`}>
+      <div className={`input-text ${className} ${validationStatus}`}>
```

Comment:
it looks like the "input" class have been deleted - any particular reason? also the validation status will show "null" as a class name if validation is null, hence the `|| '' `